src/app/components/ScheduleSummary/ScheduleSummary.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import './ScheduleSummary.css';
+import { Glyphicon } from 'react-bootstrap';
+
+export enum StatusEnum {
+    GOOD,
+    BAD,
+    WARNING,
+    EMPTY,
+    HIDDEN
+}
+export interface ScheduleSummaryProps {
+    weekStatus: {
+        sunday?: StatusEnum;
+        monday?: StatusEnum;
+        tuesday?: StatusEnum;
+        wednesday?: StatusEnum;
+        thursday?: StatusEnum;
+        friday?: StatusEnum;
+        saturday?: StatusEnum;
+    };
+}
+export default class ScheduleSummary extends React.PureComponent<ScheduleSummaryProps, {}> {
+    getDayInfo(status: StatusEnum, day: string) {
+        switch (status) {
+            case StatusEnum.GOOD:
+                return (
+                    <div className="schedule-summary-day good-day non-empty-day">
+                        <Glyphicon glyph="ok" />
+                    </div>
+                );
+            case StatusEnum.BAD:
+                return (
+                    <div className="schedule-summary-day bad-day non-empty-day">
+                        <Glyphicon glyph="remove" />
+                    </div>
+                );
+            case StatusEnum.WARNING:
+                return (
+                    <div className="schedule-summary-day warning-day non-empty-day">
+                        <Glyphicon glyph="alert" />
+                    </div>
+                );
+            default:
+                return (
+                    <div className="schedule-summary-day empty-day">
+                        {day.charAt(0).toUpperCase()}
+                    </div>
+                );
+        }
+    }
+
+    render() {
+
+        const { weekStatus } = this.props;
+        const dayStatus = Object.keys(weekStatus).map(key => ({ key, status: weekStatus[key] }));
+
+        return (
+            <div className="schedule-summary-week" >
+                {dayStatus.filter(({ status }) => status !== StatusEnum.HIDDEN).map(({ status, key }) => (
```

Comment:
I wonder.. since the days are all being passed in.  I wonder if we even need the `StatusEnum.HIDDEN` value?  You just wouldn't pass in the days you didn't wan't shown.  Just a thought now that I'm looking at the interface

Reply:
I thought of that as well - since we are both thinking it probably makes sense to take it out